src/angular/6-restaurant-service/restaurant-service.md

```diff
@@ -116,22 +116,22 @@ We've done some work to create a Place My Order API for use in this app by creat
 npm install place-my-order-api --save
 ```
 
-✏️ Next make add an api script to your `package.json`
+✏️ Next add an API script to your `package.json`
 
 ```js
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
+    "watch": "ng build --watch --configuration development",
```

Comment:
wait, why is this needed?

Reply:
this is created with a new angular app (I have updated files to match a new default ng new)